### ISSUE TITLE
Peer to Peer communication [IDEA]

### ISSUE THREAD

**Issue by lefuturiste** [LIST-LUXEMBOURG/KniwwelinoLib#9]
Like microbit with the radio peer to peer communication, it would be great if the kniwwelino could communicate in pair using their wifi capatibility (for exemple without a wifi router).

So It's would be great to have a feature in the lib to create a simple pair communication between two devices.

To discuss...

**Comment by beckerichmika** [comment]
The esp8266 can't do P2P communications, as it is WiFi-only, meaning that you need an AP to do any communication. Communication exclusively works in a client-server configuration.

**Comment by lazyzero** [comment]
Not really correct for the ESP, as it could be AP and client and mesh and espnow is possible as well. But you should come over to https://github.com/kniwwelino/KniwwelinoLib The place here is the deprecated version.

**Comment by beckerichmika** [comment]
Seriously? I always read that mesh was officially only available on the ESP32 platform or by using hacky implementations by other users. If the project moved, then please consider archiving this repo and putting a link into the README for the correct repo, as it is among the first results of at least two search engines.

**Comment by lazyzero** [comment]
Yes, it is possible that the mesh is no official implementation from Espressif. I only used ESPnow on ESP8266 to have AP-less communication, and that works quiet well. I use it for sensors that are most of the time in deep-sleep, and when they awake they send their measurement to an ESP8266 playing the gateway.

You are right we should archive this repo and link the official one. 
@step-team-jf and @LIST-ACCOUNT can you archive the repo and put a link to https://github.com/kniwwelino/KniwwelinoLib
